src/test/claimtriebranching_tests.cpp

```diff
@@ -457,7 +468,7 @@ BOOST_AUTO_TEST_CASE(claimtriebranching_spend_claim)
         check claim with more support wins
         check support delay 
 */
-BOOST_AUTO_TEST_CASE(claimtriebranching_support)
+BOOST_AUTO_TEST_CASE(support_test)
 {
```

Comment:
The previous test names were far more descriptive when tracking down logged errors.  It described module and the test name, not a name with "_test" appended (which is non-informative, given that all unit tests are tests).